app/api/v1/endpoints/chat.py

```
@@ -0,0 +1,21 @@
+
+@router.post("/chat", response_model=ChatResponse)
+async def chat_endpoint(
+    request: ChatRequest,
+    chat_service: ChatService = Depends(get_chat_service)
+):
+    """Handle chat requests."""
+    try:
+        response = await chat_service.process_chat(request)
+        return response
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)